.github/workflows/build-images-base.yaml

```diff
@@ -32,16 +46,6 @@ jobs:
     steps:
       - name: Check out code
         uses: actions/checkout@v3
-      - name: Add latest tag
-        if: ${{ github.ref_name == env.MAIN_BRANCH_NAME }}
-        id: add-latest-tag
-        run: |
-          echo "IMG_TAGS=latest ${{ env.IMG_TAGS }}" >> $GITHUB_ENV
-      - name: Add branch tag
-        if: ${{ github.ref_name != env.MAIN_BRANCH_NAME }}
-        id: add-branch-tag
-        run: |
-          echo "IMG_TAGS=${GITHUB_REF_NAME/\//-} ${{ env.IMG_TAGS }}" >> $GITHUB_ENV
```

Comment:
Is this really what we want? This will break with the single manifest link between builds.

Reply:
I don't see how we can add the expire label for different types of builds unless we do this since latest & SHA builds are usually the same on merge to main. 

Branch + SHA build would only happen if the workflow is triggered manually currently.